src/oscar/templates/oscar/partials/nav_accounts.html

```diff
@@ -29,9 +30,11 @@
                     <button class="btn btn-default" type="submit" data-loading-text="{% trans 'Submitting...' %}">{% trans "Go" %}</button>
                 </form>
             {% endif %}
+            {% endblock %}
             <ul class="nav navbar-nav navbar-right">
                 {% if user.is_authenticated %}
-                    <li>
+                {% block nav_account_navbar_authenticated %}
+                <li>
```

Comment:
Please can you fix the indentation here? The `<li>` should be indented one more level.